.github/workflows/ci.yaml

```diff
@@ -207,11 +207,25 @@ jobs:
         working-directory: ./editors/code
         if: needs.changes.outputs.typescript == 'true'
 
+  typo-check:
+    name: Typo Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      FORCE_COLOR: 1
+      TYPOS_VERSION: v1.18.0
+    steps:
+      - uses: actions/checkout@v4
+      - run: curl -LsSf https://github.com/crate-ci/typos/releases/download/$TYPOS_VERSION/typos-$TYPOS_VERSION-x86_64-unknown-linux-musl.tar.gz | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
Let's give this a name, otherwise it looks like:

![image](https://github.com/rust-lang/rust-analyzer/assets/308347/444f9371-dfca-4309-842c-a765568f1b77)